dev/tidb-in-kubernetes/deploy/gcp-gke.md

```diff
@@ -68,14 +68,14 @@ gcloud services enable cloudresourcemanager.googleapis.com \
 
 ```bash
 # 将 GCP_REGION 替换为你的 GCP Region。
-echo GCP_REGION=us-west1 >> terraform.tfvars
+echo GCP_REGION="us-west1" >> terraform.tfvars
```

Comment:
The value of the variable must be enclosed in quotes

Reply:
Quote or no quote is the same in shell.

Reply:
done.